cupy/core/core.pyx

```diff
@@ -1708,7 +1710,7 @@ cpdef ndarray array(obj, dtype=None, bint copy=True, order='K',
         else:
             a = src.copy(order=order).astype(dtype, copy=False)
 
-        ndim = a._shape.size()
+        ndim = len(a._shape)
```

Comment:
Is there any reason of this change? Is `len` faster?

Reply:
It is because I saw that the compiler complains `a._shape` doesn't have `size()` method (maybe it fails to infer `_shape` as a tuple). I now checked again and `.size()` works... It is strange. Anyway, I'll fix it.